README.md

```diff
@@ -4,10 +4,18 @@ CLI and SDK for interacting with Substra platform.
 
 ## Table of contents
 
-- [Install](#install)
-- [Usage](#usage)
-- [Documentation](#documentation)
-- [Contributing](#contributing)
+- [!Substra](#substra)
+  - [Table of contents](#table-of-contents)
+  - [Install](#install)
+  - [Usage](#usage)
+    - [CLI](#cli)
+    - [SDK](#sdk)
+  - [Documentation](#documentation)
+  - [Examples](#examples)
+  - [Contributing](#contributing)
+    - [Setup](#setup)
+    - [Documentation](#documentation-1)
+    - [Deploy](#deploy)
```

Comment:
This change seems not related to the PR scope and I'm not sure it's good idea to:
- nest all items inside `Substra`
- add the table of contents
- expand the section `Usage` and `Contributing`

Reply:
This change was automatically performed by my IDE to be honest. I pushed it because it updated a table of content that existed but was incomplete. I will undo the change so it stay simple